docs/migrating-from-0.5.md

```diff
@@ -12,6 +12,30 @@ To install the CLI:
 npm i -g genkit-cli
 ```
 
+Some changes have been made to the `genkit start` command:
+
+```posix-terminal
+
+// Starts your genkit application code + Dev UI together
+genkit start -- <language> <command>
```

Comment:
i think <language> <command> could be misleading. maybe combine into <start command> or something like that